src/Perpetuum/Perpetuum.csproj

```diff
@@ -1207,7 +1213,6 @@
     <Folder Include="Properties\" />
     <Folder Include="Services\ProductionEngine\Methods\" />
     <Folder Include="Services\Relay\RequestHandlers\" />
-    <Folder Include="Comparers\" />
```

Comment:
Where did this folder go? iirc it's what contained the `IEqualityComparer` for the timed player object

Reply:
Addressed in https://github.com/OpenPerpetuum/PerpetuumServer/pull/193